fineract-core/src/main/java/org/apache/fineract/portfolio/savings/data/SavingsAccountTransactionData.java

```diff
@@ -795,7 +797,7 @@ public boolean isReversalTransaction() {
     }
 
     public boolean isManualTransaction() {
-        return isManualTransaction;
+        return isManualTransaction != null;
```

Comment:
This can be instantiated and have a value of false, did you mean to return the actual object if not null?

Reply:
Updated the code to check for null values and always return true or false